stix_shifter_modules/onelogin/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,61 @@
+{
+  "user-account": {
+    "fields": {
+      "user_id": ["user_id"],
+      "display_name": ["user_name"]
+    }
+  },
+  "ipv4-addr": {
+    "fields": {
+      "value": ["ipaddr"]
+    }
+  },
+
+  "x-onelogin-finding": {
+    "fields": {
+      "client_id": ["client_id"],
+      "directory_id": ["directory_id"],
+      "event_type_id": ["event_type_id"],
+      "unique_id": ["id"],
+      "resolution": ["resolution"],
+      "since": ["since"],
+      "until": ["until"],
+      "proxy_ip": ["proxy_ip"],
```

Comment:
This should be in a `ipv4-addr` and `ipv6-addr` object. In general, you should be mapping to standard STIX objects when possible. Relying so heavily on custom objects limits the ability of stix-shifter to conduct federated search across multiple data sources.

Reply:
Yes  @delliott90  moved  `proxy_ip` to `ipv4-addr` . Since OneLogin not support `OR` comparison operator  added as custom field `x_proxy_ip`